.github/workflows/warn-on-editing-requirements.yml

```diff
@@ -0,0 +1,26 @@
+name: Warn on editing requirements
+
+on:
+  pull_request_target:
+    paths:
+      - setup.py
+      - requirements/skinny-requirements.yaml
```

Comment:
we might want to change the directory/file structure to make it easier to detect requirement updates.

Reply:
does this current way miss some cases? it seemed relatively straightforward to me

Reply:
I want to avoid false positives. For example, the following change shouldn't trigger this workflow.

```diff
diff --git a/setup.py b/setup.py
index c37b1c2e05..6eb939d8ad 100644
--- a/setup.py
+++ b/setup.py
@@ -96,7 +96,7 @@ class ListDependencies(Command):
         print("\n".join(dependencies))
 
 
-MINIMUM_SUPPORTED_PYTHON_VERSION = "3.8"
+MINIMUM_SUPPORTED_PYTHON_VERSION = "3.9"
 
 
 class MinPythonVersion(Command):
```

Reply:
Ah yeah that makes sense. I'll change this workflow to just require an approval from a maintainer, so even if false positives happen it shouldn't cause a lot of friction